<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Tomeu Borras</author>
 <class>VehiculoBase</class>
 <widget class="QWidget" name="VehiculoBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nuevo vehiculo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/BfBulmaFact32x32/images/pendientes/iconbulmafact.xpm</normaloff>:/BfBulmaFact32x32/images/pendientes/iconbulmafact.xpm</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="pixmapLabel1">
       <property name="pixmap">
        <pixmap resource="../../../bulmages_oxygen.qrc">:/BfBulmaFact48x48/images/png/48x48/i_bulmages_boton_mas.48x48.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_guardar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Guardar</string>
          </property>
          <property name="whatsThis">
           <string>Guardar</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Genericos32x32/images/pendientes/i_save.xpm</normaloff>:/Genericos32x32/images/pendientes/i_save.xpm</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Borrar</string>
          </property>
          <property name="whatsThis">
           <string>Borrar</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Genericos32x32/images/pendientes/i_dele.png</normaloff>:/Genericos32x32/images/pendientes/i_dele.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_imprimir">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Imprimir</string>
          </property>
          <property name="whatsThis">
           <string>Imprimir</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Genericos32x32/images/pendientes/i_filePrint.xpm</normaloff>:/Genericos32x32/images/pendientes/i_filePrint.xpm</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>146</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text">
           <string>Matricula:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BfBuscarReferencia" name="mui_matriculavehiculo"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="mui_detalles">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Imprimir pedido</string>
       </property>
       <property name="whatsThis">
        <string>Imprimir pedido</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff/>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="3" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="BlDateSearch" name="mui_fechacompravehiculo" native="true"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="mui_preciovehiculo"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mui_modelovehiculo"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mui_marcavehiculo"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Marca:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Precio de compra:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Modelo:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Fecha de compra:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="BfSubForm" name="mui_listadomantvehiculo"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>419</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2"/>
 <customwidgets>
  <customwidget>
   <class>BfSubForm</class>
   <extends>QTableWidget</extends>
   <header>bfsubform.h</header>
  </customwidget>
  <customwidget>
   <class>BlDateSearch</class>
   <extends>QWidget</extends>
   <header>bldatesearch.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BfBuscarReferencia</class>
   <extends>QLineEdit</extends>
   <header>bfbuscarreferencia.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages_oxygen.qrc"/>
  <include location="../../../bulmages.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mui_detalles</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
